4-projects/modules/single_project/main.tf

```diff
@@ -20,22 +20,28 @@ locals {
 
 module "project" {
   source                      = "terraform-google-modules/project-factory/google"
-  version                     = "~> 7.0"
+  version                     = "~> 8.0"
   random_project_id           = "true"
   impersonate_service_account = var.impersonate_service_account
   activate_apis               = var.activate_apis
   name                        = "${var.project_prefix}-${var.environment}"
```

Comment:
we can remove environment from the name here as we are defining as env_code is already in the project name